src/scopes.py

```diff
@@ -19,12 +20,13 @@ def outer():
     y = 120
 
     def inner():
+        global y
```

Comment:
This works, but there's a way to do it with the `nonlocal` keyword while keeping the print statement inside the function https://www.w3schools.com/python/ref_keyword_nonlocal.asp